tests/integration/components/phone-input-test.js

```diff
@@ -128,4 +128,20 @@ module('Integration | Component | phone-input', function(hooks) {
     );
     assert.ok(find('input').disabled);
   });
+
+  test('can prevent the dropdown', async function(assert) {
+    assert.expect(1);
+
+    await this.owner.lookup('service:phone-input').load();
+
+    this.set('update', value => {
+      this.set('allowDropdown', value);
```

Comment:
Sorry I was not clear enough on the update method. Here you need to update the number, not the allowDropdown option.

Reply:
Or you might be able to remove it at all